### DISCUSSION TITLE
Bootstrap 5 Update?

### DISCUSSION THREAD

**Discussion by MrAlexWinkler** [danirus/django-comments-xtd#366] [Q&A]
Currently Django-comments-xtd is setup for bootstrap 4 but is there any plans for expanding to bootstrap 5?

**Comment by danirus** [comment]
The plan is to get rid of bootstrap as a dependency altogether.
The v3 (already implemented in [branch v3.0.0](https://github.com/danirus/django-comments-xtd/tree/rel-3.0.0) but missing a few JavaScript tests and the new documentation), already gets rid of Bootstrap and ReactJS. It uses a vanilla JavaScript plugin.
If you are interested on giving v3 a try, I can give you instructions here on how to set up the two demo projects: `demos/project-quotes` and `demos/project-stories`. It's barely explained in their README.md files.

I'm planning to write another demo project to use bootstrap into v3. But it's not there yet.

**Comment by MrAlexWinkler** [comment]
Hey @danirus thanks for the message. I think that makes a lot of sense. 

Yes please, would be interested in testing it out. Atm we're about to release v2.9.5 into our project, but we can start playing around with the newer version for an upcoming release. No rush!

**Comment by gamboz** [comment]
New development seem to be here: https://pypi.org/project/django-comments-ink/
